### DISCUSSION TITLE
Emoji in details summary markdown (PR messages)

### DISCUSSION THREAD

**Discussion by github-samuel-rats** [community/community#77863] [Pull Requests]
### Select Topic Area

Bug

### Body

The following `details` block does not display emojis properly. It worked last week. 

<details><summary>:x: emoji is not displayed in this message: <strong>0</strong>, <strong>0</strong>, <strong>0</strong>.</summary>

test
</details>

<details><summary>:white_check_mark: emoji is properly displayed in this message: 0, 0, 0.</summary>

test
</details>

Seems to be related to nested balise.

Screenshot on my end: 
![Screenshot from 2023-12-04 14-45-20](https://github.com/community/community/assets/109607572/884bfc19-ca7f-4afc-900f-5c99dea1114d)

**Comment by DenverCoder1** [comment]
What is the markup you are using to create the details summary? When I try it like this, it appears to work

```html
<details>
<summary>:x: emoji is not displayed in this message: 0, 0, 0.</summary>
test
</details>
```

<details>
<summary>:x: emoji is not displayed in this message: 0, 0, 0.</summary>
test
</details>

Screenshot:

![image](https://github.com/community/community/assets/20955511/986b9b2f-25dd-40b6-a9b3-466f33e3f604)

**Comment by github-samuel-rats** [comment]
```
<details><summary>:x: emoji is not displayed in this message: <strong>0</strong>, <strong>0</strong>, <strong>0</strong>.</summary>

test
</details>
```

**Reply by DenverCoder1** [reply]
Interesting, it seems like the use of `<strong>` is making a difference to how the emoji renders for some reason.

At least as a temporary workaround, it is possible to use the emoji symbol `❌` instead of the shortcode.

```html
<details><summary>❌ emoji is not displayed in this message: <strong>0</strong>, <strong>0</strong>, <strong>0</strong>.</summary>

test
</details>
```

**Comment by CCedricYoung** [comment]
I observed this too, that emoji's which happened sometime during/after the PR service incident last week.

It seems to be something to do with the parsing mis-detecting start/end or non-emjoi sequences. 
For example:
```
1. :white_check_mark:  end with a colon:
```
renders as 
![image](https://github.com/community/community/assets/4129217/fb6cb70c-e4de-4607-a563-084cb681874a)

But 
```
1. :white_check_mark:  don't end with a colon
```
renders the emoji correctly
![image](https://github.com/community/community/assets/4129217/a96af622-4712-47ad-bf9c-b5887d47c230)

**Reply by CCedricYoung** [reply]
This isn't just limited to a hanging `:` at the end, but if you use MD links, or inline-code, etc

**Reply by CCedricYoung** [reply]
btw I'm just a consumer of GitHub PR's that is affected by this as well.

**Comment by dakshithadissanayaka** [comment]
It appears that you're encountering an issue with the display of emojis in a particular context. If you're dealing with an application or platform, it's possible that there might be changes or issues on their end. Here are some steps you can consider to address or report the issue:

1. **Check Platform/Tool Updates:**
   - Ensure that the platform or tool you are using is up to date. Sometimes, updates or changes in the platform can affect the rendering of emojis.

2. **Browser or Application Cache:**
   - Clear the cache of the browser or application you are using. Cached data might be causing display issues.

3. **Check Browser Compatibility:**
   - If you are using a web application, try a different browser to see if the issue persists. Some browsers might handle emojis differently.

4. **Platform/Tool Support:**
   - Check the documentation or support resources for the platform or tool you are using. There might be information about known issues or changes that could be affecting emoji rendering.

5. **Report the Issue:**
   - If the problem persists, consider reporting the issue to the support or development team of the platform or tool. Provide them with details about your environment, the issue you're facing, and any relevant screenshots.

6. **Test in Different Contexts:**
   - If possible, test the emoji rendering in different contexts or platforms to see if the issue is specific to a certain environment.

Without more specific details about the platform or tool you are using, it's challenging to provide a more targeted solution. If you can provide more information about the application, website, or tool where you're experiencing this issue, I might be able to offer more tailored advice.

**Reply by CCedricYoung** [reply]
hey @dakshithadissanayaka,
For reporting the issue, where exactly does one do that?

**Comment by alethenorio** [comment]
It seems that simply wrapping a piece of text in between 2 emojis is enough to show the issue

**Compare**

:green_circle: Some Text

**And**

:green_circle: Some Text :green_circle:

This was working until a couple of days ago

**Comment by github-samuel-rats** [comment]
Looks like everything is back to normal, thanks for the fix!
